docs/release/0.12.0.rst

```diff
@@ -0,0 +1,81 @@
+============================
+pyFFTW v0.12.0 Release Notes
+============================
+
+We are happy to announce the release of pyFFTW v0.12.0.
+
+The highlight of this release is the addition of interfaces for the
+scipy.fft module that was introduced with SciPy 1.4. This release is now
+compatibile with SciPy 1.4. This release supports Python 2.7 and
+3.5-3.8.
+
+pyFFTW is a pythonic wrapper around FFTW 3, the speedy FFT library. The
+ultimate aim is to present a unified interface for all the possible transforms
+that FFTW can perform.
+
+Both the complex DFT and the real DFT are supported, as well as on arbitrary
+axes of abitrary shaped and strided arrays. Operating FFTW in multithreaded
+mode is supported.
+
+pyFFTW implements the numpy and scipy fft interfaces in order for users to take
+advantage of the speed of FFTW with minimal code modifications. A dask fft
+interface is provided as a drop-in replacement for the equivalent module in
+dask.
+
+
+New features
+============
+
+scipy.fft interface
+-------------------
+This interface operates like the existing scipy.fftpack interface, but
+matches the API of the newer scipy.fft module introduced in SciPy 1.4.
+
+
+Bugs Fixed
+==========
+
+The test suite was updated to be compatible with more recent dask (#278).
+
+The Cython variable _N was renamed to avoid a name conflict with a preprocessor
+token on some platforms (#259).
+
+
+Other changes
+=============
+
+Python 3.4 support has been dropped. pyFFTW now supports Python 2.7 and 3.5-3.8.
+
+
+Authors
+=======
+
+* Peter Bell +
+* Gregory R. Lee
+* Stefan Peterson +
+* DWesl +
+
+A total of 4 people contributed PRs to this release.
+People with a "+" by their names contributed a patch for the first time.
+
+John Kirkham and Henry Gommersall also contributed by helping review PRs.
```

Comment:
Very minor, but it's "Gomersall" (one m)

Reply:
Thanks, sorry about that!